bin/node_modules/nopt/.npmignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Also - do you not need to check in the "abbrev" module, since nopt depends on it?